src/main/resources/application.yml

```
@@ -5,3 +5,7 @@ spring:
     mongodb:
       uri: mongodb+srv://myungmoondev:${MONGO_PW}@luv.fr7ah.mongodb.net/?retryWrites=true&w=majority&appName=luv
       database: ${MONGO_DBNAME}
+cloudflare:
+  images:
+    account-id: ${CLOUDFLARE_ACCOUNT_ID}
+    api-token: ${CLOUDFLARE_API_KEY}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)